com.skype.Client.appdata.xml

```diff
@@ -18,10 +18,10 @@
     </screenshot>
```

Comment:
Change `<screenshot>` to `<screenshot type="default">` , github doesn't let me comment there.